stdlib/public/Concurrency/CooperativeExecutor.swift

```diff
@@ -70,6 +70,21 @@ extension ExecutorJob {
     }
   }
 
+  fileprivate var cooperativeExecutorSequence: UInt {
```

Comment:
Maybe Sequence "Number"? I see the comment below that explains how we use it, maybe just the name could include number here

```suggestion
  fileprivate var cooperativeExecutorSequenceNr: UInt {
```

Reply:
IMO `Sequence` is a clear enough name on its own — it can't really be anything other than a sequence number, since it's a variable that holds a number, and there _is_ a comment explaining what it's for. Plus if I put `Nr` on the end, someone else will tell me it needs to be `Number` and it's already a pretty long name…

Reply:
Meh, okey :) Let's stick with this, not a big deal, it is private anyway 👍